src/elements/content-sidebar/activity-feed/comment-form/CommentForm.js

```diff
@@ -27,7 +27,7 @@ type Props = {
     contactsLoaded?: boolean,
     createComment?: Function,
     entityId?: string,
-    getAvatarUrl: GetAvatarUrlCallback,
+    getAvatarUrl?: GetAvatarUrlCallback,
```

Comment:
Curious, what do we do when there is no avatar url? Do we show anything in place of the avatar?

Reply:
In this case we wouldn't show anything. Previously we would require the cb, but the reply form doesn't need an avatar so I made it optional, and to just not show the avatar if there's no cb passed in.

Reply:
Actually looks like `Avatar.js` doesn't require a `getAvatarUrl` in the component. So I will remove that boolean from the conditional.